prometheus/src/main/java/org/opensearch/sql/prometheus/client/PrometheusClientImpl.java

```diff
@@ -82,7 +81,9 @@ public Map<String, List<MetricMetadata>> getAllMetrics() throws IOException {
   private List<String> toListOfStrings(JSONArray array) {
     List<String> result = new ArrayList<>();
     for (int i = 0; i < array.length(); i++) {
-      result.add(array.optString(i));
+      if (!"__name__".equals(array.optString(i))) {
+        result.add(array.optString(i));
+      }
```

Comment:
Could you add comment? cannot figure out what does this mean...

Reply:
Doing that.